doc/user_guide/user_guide.md

```diff
@@ -290,36 +257,24 @@ severely. Available task_types are the same as the names of our available UDFs,
 `translation` and`zero_shot_classification`.    
 
 ### 2. Model Uploader Script
-You can invoke the python script as below which allows to download the transformer 
-models from The Hugging Face hub to the local filesystem, and then from there to the BucketFS.
-
-
-#### List of options
-
-The table below lists all available options. It shows which ones are applicable for On-Prem and for SaaS backends.
-Unless stated otherwise in the comments column, the option is required for either or both backends.
-
-| Option name                  | On-Prem | SaaS | Comment                                         |
-|:-----------------------------|:-------:|:----:|:------------------------------------------------|
-| bucketfs-name                |   [x]   |      |                                                 |
-| bucketfs-host                |   [x]   |      |                                                 |
-| bucketfs-port                |   [x]   |      |                                                 |
-| bucketfs-user                |   [x]   |      |                                                 |
-| bucketfs-password            |   [x]   |      | Env. [BUCKETFS_PASSWORD]                        |
-| bucketfs-use-https           |   [x]   |      | Optional boolean, defaults to True              |
-| bucket                       |   [x]   |      |                                                 |
-| saas-url                     |         | [x]  |                                                 |
-| saas-account-id              |         | [x]  | Env. [SAAS_ACCOUNT_ID]                          |
-| saas-database-id             |         | [x]  | Optional, Env. [SAAS_DATABASE_ID]               |
-| saas-database-name           |         | [x]  | Optional, provide if the database_id is unknown |
-| saas-token                   |         | [x]  | Env. [SAAS_TOKEN]                               |
-| model-name                   |   [x]   | [x]  |                                                 |
-| path-in-bucket               |   [x]   | [x]  | Root location in the bucket for all models      |
-| sub-dir                      |   [x]   | [x]  | Sub-directory where this model should be stored |
-| task_type                    |   [x]   | [x]  | Name of the task you want to use the model for  |
-| token                        |   [x]   | [x]  | Huggingface token (needed for private models)   |
-| [no_]use-ssl-cert-validation |   [x]   | [x]  | Optional boolean, defaults to True              |
+You can invoke the Python script below which downloads the transformer 
+models from The Hugging Face hub to the local filesystem, then to the BucketFS.
```

Comment:
I prefer the existing sentence here. the script exists specifically for people who do not want to connect from their DB directly to the huggingface hub, but rather want to only connect to the db from their own machine. i think this should be empathized.

Reply:
The existing sentence doesn't say this either. I will add your explanations of who this script is for.

Reply:
Actually, we are already saying this in the starting paragraph of the **Store Models in BucketFS** section.

"The Model Downloader UDF is the simpler option, but if you do not want to connect your Exasol Database
directly to the internet, the Model Uploader Script is an option for you."

There is no need to repeat this.